### PR TITLE
City administration

### PR DESCRIPTION
**Pull request by IvanSultanov**
Hello, Yauheni.
Could you please try to merge my part.
I added Db seeding (_Clear Db_ and _Seed Db_ via appsettings works fine).
Thank you in advance.

### CODE REVIEW

File: UtopiaCity/Controllers/Airport/FlightController.cs [hunk 1]
```diff
@@ -1,124 +0,0 @@
-﻿using Microsoft.AspNetCore.Mvc;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using UtopiaCity.Data;
-using UtopiaCity.Models.Airport;
-using UtopiaCity.Services.Airport;
-
-namespace UtopiaCity.Controllers.Airport
```
Comment (crocomoth) [inline UtopiaCity/Controllers/Airport/FlightController.cs:10]: It shouldn't be deleted I think?

File: UtopiaCity/Controllers/Airport/WeatherReportController.cs [hunk 2]
```diff
@@ -1,130 +0,0 @@
-﻿using Microsoft.AspNetCore.Mvc;
```
Comment (crocomoth) [inline UtopiaCity/Controllers/Airport/WeatherReportController.cs:1]: Same here, not your code as I see

File: UtopiaCity/Controllers/CityAdministration/RersidentAccountsController.cs [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using UtopiaCity.Data;
+using UtopiaCity.Models.CityAdministration.ResidentAccount;
+using UtopiaCity.Services.CityAdministration;
+
+namespace UtopiaCity.Controllers.CityAdministration
+{
+    public class RersidentAccountsController : Controller
```
Comment (crocomoth) [inline UtopiaCity/Controllers/CityAdministration/RersidentAccountsController.cs]: Typo

File: UtopiaCity/Controllers/Sport/SportComplexController.cs [hunk 4]
```diff
@@ -1,93 +0,0 @@
-﻿using Microsoft.AspNetCore.Mvc;
```
Comment (crocomoth) [inline UtopiaCity/Controllers/Sport/SportComplexController.cs:1]: Not your code, again?

File: UtopiaCity/Data/ApplicationDbContext.cs [hunk 5]
```diff
@@ -14,11 +13,6 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         }
 
         public DbSet<EmergencyReport> EmergencyReport { get; set; }
-        
-        public DbSet<SportComplex> SportComplex { get; set; }
```
Comment (crocomoth) [inline UtopiaCity/Data/ApplicationDbContext.cs:18]: not your code

**Review by crocomoth — CHANGES_REQUESTED** [review]
Okay, seems like you had merge conflicts and resolved them incorrectly, Lets check this on lecture together, you are not the only one for sure

File: UtopiaCity/Controllers/Airport/FlightController.cs (hunk 1 above)
Comment (IvanSultanov) [reply UtopiaCity/Controllers/Airport/FlightController.cs:10]: No, It's not my code.

File: UtopiaCity/Controllers/Airport/WeatherReportController.cs (hunk 2 above)
Comment (IvanSultanov) [reply UtopiaCity/Controllers/Airport/WeatherReportController.cs:1]: Correct

File: UtopiaCity/Controllers/Sport/SportComplexController.cs (hunk 4 above)
Comment (IvanSultanov) [reply UtopiaCity/Controllers/Sport/SportComplexController.cs:1]: Correct.

File: UtopiaCity/Data/ApplicationDbContext.cs (hunk 5 above)
Comment (IvanSultanov) [reply UtopiaCity/Data/ApplicationDbContext.cs:18]: Correct.

File: UtopiaCity/Controllers/CityAdministration/RersidentAccountsController.cs (hunk 3 above)
Comment (IvanSultanov) [reply UtopiaCity/Controllers/CityAdministration/RersidentAccountsController.cs]: Accepted.

**Comment by IvanSultanov** [comment]
> Okay, seems like you had merge conflicts and resolved them incorrectly, Lets check this on lecture together, you are not the only one for sure

I Just added my code to the dev branch and made pull request, which (obviously) caused a lot of conflicts. As I understand now, first I should merge origin/dev into my branch (CityAdministartion) resolving all conflicts and only then make pull request to origin/dev. Do I correct understand?

**Pull request closed without merging**